tests/probes_test.go

```diff
@@ -140,12 +128,32 @@ var _ = Describe("[ref_id:1182]Probes", func() {
 		tcpProbe := createTCPProbe(period, initialSeconds, port)
 		httpProbe := createHTTPProbe(period, initialSeconds, port)
 
-		table.DescribeTable("should not fail the VMI", func(livenessProbe *v12.Probe, serverStarter func(vmi *v12.VirtualMachineInstance, port int)) {
-			By("Specifying a VMI with a liveness probe")
-			vmi := createReadyCirrosVMIWithLivenessProbe(virtClient, livenessProbe)
+		table.DescribeTable("should not fail the VMI", func(livenessProbe *v12.Probe, IPFamily v1.IPFamily, serverStarter func(vmi *v12.VirtualMachineInstance, port int)) {
 
-			By("Starting the server inside the VMI")
-			serverStarter(vmi, 1500)
+			if IPFamily == v1.IPv6Protocol {
+				libnet.SkipWhenNotDualStackCluster(virtClient)
+			}
+
+			if IPFamily != "" {
```

Comment:
since ipv4 case works with tests.StartHTTPServer I suggest we only use the pod-sidekick in ipv6 case:
```
if IPFamily == v1.IPv6Protocol {
```

Reply:
Done.